deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml

```diff
@@ -1372,9 +1372,17 @@ spec:
                 - tag
                 type: object
               type: array
+            license:
+              properties:
+                accept:
+                  type: boolean
+              required:
+                - accept
```

Comment:
does [this](https://github.com/IBM/ibm-block-csi-operator/pull/135#issuecomment-790055884) affect our seamless upgrade/ui?
do we need to update anything in the UG installation steps?
do we need to update our CR yaml? is there an implicit default `false` value?

Reply:
Also, do we need to update the OSSC page?

Reply:
It's passing sanity with CERTIFIED_OLM.
if it's needs anything else, Im not aware of it. it might come up with an upgrade test.
Barak asks to put it in and to revert in case of any problem.